util/types/src/core/service.rs

```diff
@@ -29,3 +31,57 @@ impl<A, R> Request<A, R> {
         response.recv().ok()
     }
 }
+
+/// Notify pool transaction entry
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct PoolTransactionEntry {
+    /// Transaction view
+    pub transaction: TransactionView,
+    /// Transaction consumed cycles
+    pub cycles: Cycle,
+    /// Transaction serialized cycles
+    pub size: usize,
+    /// Transaction fee
+    pub fee: Capacity,
+}
+
+/// TX reject message
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub enum PoolTransactionReject {
```

Comment:
```suggestion
#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
#[serde(tag = "type", content = "description")]
pub enum PoolTransactionReject {
```

So, the return looks like the follow structure:
```javascript
[
    // first item is the `PoolTransactionEntry`
    {
        "transaction": { ... /* omit */ ... }
    },
    // second item is the `PoolTransactionReject`
    {
        "type": "Resolve",
        "description": "Resolve failed Dead(OutPoint(...))"
    }
]  
```